config/bootstrap.js

```diff
@@ -11,7 +11,12 @@
 
 module.exports.bootstrap = function(cb) {
 
-  // It's very important to trigger this callback method when you are finished
-  // with the bootstrap!  (otherwise your server will never lift, since it's waiting on the bootstrap)
-  cb();
+  Video.findOne({
+    playing: true
```

Comment:
Unfortunately that's sails. .then() isn't a function on their Waterline promises.